### ISSUE TITLE
MdcSelect right text justification does not work

### ISSUE THREAD

**Issue by MarkStega** [Material-Blazor/Material.Blazor#56]
It was this way before the work I did on MdcSelect

**Comment by SQL-MisterMagoo** [comment]
Is this what you are after on this one?

![image](https://user-images.githubusercontent.com/11338430/81589921-e80ecd00-93b1-11ea-8421-0a129d9a8a25.png)

**Comment by MarkStega** [comment]
Looks correct to me -- I wasn't looking to use it, just noted that it wasn't working in the demo pae.

**Comment by simonziegler** [comment]
The dropdown now works as required, however the main <div> when no dropdown shows also needs to right align.

**Comment by SQL-MisterMagoo** [comment]
Hi
These are the issues as I see them - could you perhaps give me your thoughts?

Material Design doesn't appear to allow for right alignment of text in inputs.
Setting text-align works in some places (filled select) but not others (outlined select) due to the standard css for mdc.

When you have an outlined select, it sets `display: flex` which then defaults to `justify-content: flex-start`, and overrides `text-align: right`

``` css
.mdc-select--outlined .mdc-select__selected-text {
    padding-left: 16px;
    padding-right: 52px;

    display: flex;

    padding-top: 14px;
    padding-bottom: 12px;
    border: none;
    background-color: transparent;
    z-index: 1
}
```

You could override this in your own styles - but there is a little issue with Webcompiler/bundler, which I suspect you encountered where it doesn't handle `@using` statements in the middle of a bundle (it should move them to the start but doesn't), so you are bundling your styles *before* the standard mdc ones. This prevents you overriding any styles from your CSS.

That can be overcome by moving the `@using` for google fonts out of styles.scss and swapping the order of the SCSS files in bundleconfig.json.

Now, you have a situation where right align works for all selects.

![image](https://user-images.githubusercontent.com/11338430/81679300-0aecc000-944a-11ea-86a6-a62fb05488ef.png)

However, as you can see, there are other text elements that are not right-aligned - the placeholder / label text(s).

![image](https://user-images.githubusercontent.com/11338430/81679719-569f6980-944a-11ea-892b-0f6913fd25df.png)

Would you want to change those also?

I have sent a draft PR for you to examine as this is now less trivial.

**Comment by simonziegler** [comment]
I believe that you have struck the right balance here. In an LTR language floating labels should definitely be left aligned, and I'm not worried that the non-floating labels also being left aligned. The purpose for right aligned selects is for numeric selection, and for now I don't envisage (a) using it more than a couple of times and (b) having an empty default. So this looks great as it is.

Regarding switching the bundling order and including fonts, this also looks ideal. I looked through the draft PR and am very happy to merge and close this issue.

Thank you

**Comment by simonziegler** [comment]
Closed by #87
